apps/prairielearn/src/tests/variantAccess.test.ts

```diff
@@ -147,7 +147,9 @@ async function assertVariantAccess({
 
   // Test access to a rendered submission.
   const submissionUrl = `${siteUrl}${questionBasePath}${isStudentRoute ? '' : '/preview'}/variant/${variantId}/submission/${submissionId}`;
+  console.log('C', submissionUrl);
   const submissionRes = await fetch(submissionUrl);
+  console.log('after C');
```

Comment:
👀 leftover debugging logs?

Reply:
Fixed in 6d3ab0659925d5cb29dd02ca81164ee829466c43.